contracts/feature-tests/composability/forwarder-queue/src/forwarder_queue.rs

```diff
@@ -32,6 +32,9 @@ pub trait ForwarderQueue {
     #[proxy]
     fn self_proxy(&self, to: ManagedAddress) -> crate::Proxy<Self::Api>;
```

Comment:
No more need for the self proxy, we will allow it to call anything in a raw fashion.